### PR TITLE
ci: add subs-disabled test

### PR DESCRIPTION
**Pull request by iartemiev**
Adding e2e test for DataStore working with disabled subscriptions to the pipeline

Successful run here: https://app.circleci.com/pipelines/github/aws-amplify/amplify-js/7443/workflows/b62d22a8-1a77-428f-ae1a-b9b7c5415846

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -551,6 +551,24 @@ jobs:
...
+    parameters:
+      browser:
+        type: string
+    executor: js-test-executor
+    <<: *test_env_vars
+    working_directory: ~/amplify-js-samples-staging/samples/react/datastore/subs-disabled
+    steps:
+      - prepare_test_env
+      - integ_test_js:
+          test_name: 'React DataStore'
```
Comment (sammartinez) [inline .circleci/config.yml:565]: This doesnt mash up with anything else we have for React?

Comment (iartemiev) [reply .circleci/config.yml]: That's just a display name / label for the job that shows up in the CCI UI, but might as well call it the same thing as the test itself. Updated

File: .circleci/config.yml [hunk 2]
```diff
@@ -551,6 +551,24 @@ jobs:
...
+    parameters:
+      browser:
+        type: string
+    executor: js-test-executor
+    <<: *test_env_vars
+    working_directory: ~/amplify-js-samples-staging/samples/react/datastore/subs-disabled
+    steps:
+      - prepare_test_env
+      - integ_test_js:
+          test_name: 'DataStore - Subs Disabled'
```
Comment (sammartinez) [inline .circleci/config.yml:565]: 🥇

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Review by wlee221 — APPROVED** [review]
LGTM 👍

**Pull request merged**